build.psm1

```diff
@@ -1223,7 +1223,12 @@ function Start-PSPester {
 
     if ($Unelevate)
     {
-        $outputBufferFilePath = [System.IO.Path]::GetTempFileName()
+        if ($environment.IsWindows) {
+            $outputBufferFilePath = [System.IO.Path]::GetTempFileName()
+        }
+        else {
+            $outputBufferFilePath = (Join-Path $env:HOME $([System.IO.Path]::GetRandomFileName()))
```

Comment:
Just curious, why doesn't `[Path]::GetTempFileName` wokr for Linux/macOS?

Reply:
Ubuntu 20.04 does not seem to have the temp directory defined by default.

Reply:
I guess that's a specific issue with the build agent. Can you please add a comment about this?